README.md

```diff
@@ -44,91 +52,156 @@ const secondWord = "squeeze";
+
 ```js
-console.log()
+console.log(firstWord + " " + secondWord + " " + thirdWord + " " + fourthWord);
 ```
 
 Output a console log `The sum of 5 and 10 is 15` where the values for 5 and 10 are saved to variables, and where 15 comes from those variables being summed.
+
 ```js
 const num1 = 5;
 const num2 = 10;
 ```
 
 ### Your solution here:
+
 5.  How can we make `num3` equal to the sum of `num1` and `num2`?
+
 ```js
 // your solution here
+const num3 = num1 + num2;
 ```
-6.  Use variables `num1`, `num2` and `num3` to fill in the `console.log()` to complete the sentence: 
 
->The sum of 5 and 10 is 15
+6.  Use variables `num1`, `num2` and `num3` to fill in the `console.log()` to complete the sentence:
+
+> The sum of 5 and 10 is 15
 
 ```js
-console.log()
+console.log("The sum of " + num1 + " and " + num2 + " is " + num3);
 ```
 
 ## Comparisons
+
 By just looking at the following expressions, determine in your mind whether or not each will evaluate to true or false
+
 ```
 a) 999 > 999
-b) 999 === 999 
+b) 999 === 999
 c) 999 !== 999
 d) -5 >= -4
 e) 100 <= -100
-f) 20 + 5 < 5 
+f) 20 + 5 < 5
 g) 81 / 9 === 9
 h) 9 !== 8 + 1
 ```
+
 ### Your solution here:
+
 7.  Write `true` or `false` based on the list above
+
 ```
-a) 
-b)  
-c)
-d) 
-e) 
-f) 
-g) 
-h) 
+a) false
+b) true
+c) false
+d) false
+e) false
+f) false
+g) true
+h) false
 ```
 
 ## Conditionals
+
 Declare a variable equal to a number 0 to 100
 
 Write a conditional statement that...
+
 - If it is a multiple of 3, print “Fizz” instead of the number.
 - If it is a multiple of 5, print “Buzz” instead of the number.
 - If it is a multiple of both 3 and 5, print “FizzBuzz” instead of the number.
 - Otherwise, print the number
 
 ### Your solution here:
+
 8.  Write your javascript solution below
+
 ```js
 // your answer here
+
+num100 = 10;
+
+if (num100 % 3 === 0 && num100 % 5 === 0) {
+  console.log("FizzBuzz");
+} else if (num100 % 5 === 0) {
+  console.log("Buzz");
+} else if (num100 % 3 === 0) {
+  console.log("Fizz");
+} else {
+  console.log(num100);
+}
 ```
 
 #### BONUS
+
 9.  Research a [loop](https://javascript.info/while-for) so that your condition runs on every number from 0 to 100
+
 ```js
 // your answer here
+
+for (let i = 1; i <= 100; i++) {
+  let multipleOf_5 = i % 5 === 0;
+
+  let multipleOf_3 = i % 3 === 0;
+
+  let multipleOf_11 = i % 11 === 0;
+
+  if (multipleOf_3) {
+    console.log("Fizz");
+  } else if (multipleOf_5) {
+    console.log("Buzz");
+  } else if (multipleOf_11) {
+    console.log("FizzBuzz");
+  }
+  console.log(i);
+}
 ```
-10.  Research a [function](https://javascript.info/function-basics) so that your condition runs on every number from 0 to whatever number is passed into the function
+
+10. Research a [function](https://javascript.info/function-basics) so that your condition runs on every number from 0 to whatever number is passed into the function
+
 ```js
 // your answer here
+
+const multiple = function(num) {
```

Comment:
should've used your algorithm from question 8. also this doesn't iterate to the desired value.